lms/djangoapps/teams/static/teams/js/views/create_team.js

```diff
@@ -0,0 +1,43 @@
+;(function (define) {
+'use strict';
+
+define(['backbone',
+        'underscore',
+        'gettext',
+        'js/components/header/models/header',
+        'js/components/header/views/header',
+        'teams/js/views/edit_team'],
+       function (Backbone, _, gettext, HeaderModel, HeaderView, TeamEditView) {
+           return Backbone.View.extend({
+               initialize: function(options) {
+
+                   this.headerView = new HeaderView({
+                       model: new HeaderModel({
+                           description: gettext("Create a new team if you can't find existing teams to join, or if you would like to learn with friends you know."),
+                           title: gettext("Create a New Team"),
+                           breadcrumbs: [
+                               {
+                                   title: options.teamParams.topicName,
+                                   url: Backbone.history.location.href.replace(/\/create_new_team\/?/,  '')
```

Comment:
Ah I see. @frrrances what do you think of @andy-armstrong's idea which I mentioned above?